Makefile

```diff
@@ -0,0 +1,85 @@
+NAMESPACE := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["namespace"])')
+NAME := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["name"])')
+VERSION := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["version"])')
+MANIFEST := build/collections/ansible_collections/$(NAMESPACE)/$(NAME)/MANIFEST.json
+
+ROLES := $(wildcard roles/*)
+DOCS := $(wildcard docs/*)
+META := $(wildcard meta/*)
+PLAYBOOKS := $(wildcard playbooks/*)
+METADATA := galaxy.yml LICENSE README.md requirements.yml
+DEPENDENCIES := $(METADATA) $(foreach ROLE,$(ROLES),$(wildcard $(ROLE)/*/*)) $(DOCS) $(foreach DOC,$(DOCS),$(wildcard $(DOC)/*)) $(META) $(PLAYBOOKS) $(foreach PLAY,$(PLAYBOOKS),$(wildcard $(PLAY)/*)) $(foreach PLAY,$(PLAYBOOKS),$(wildcard $(PLAY)/*/*))
+
```

Comment:
I also got it from squeezer, but I'm not sure if I did it right, @mdellweg  can you please take a look?